Packs/Mattermost/Integrations/MattermostV2/MattermostV2.py

```diff
@@ -1200,6 +1243,39 @@ def list_channels_command(client: HTTPClient, args: dict[str, Any]) -> CommandRe
     )
 
 
+def list_private_channels_for_user_command(client: HTTPClient, args: dict[str, Any]) -> CommandResults:
+    """ Lists private channels for user """
+    team_name = args.get('team_name', client.team_name)
+    user_id = args.get('user_id', '')
+    list_private_channels = []
+
+    team_details = client.get_team_request(team_name)
+
+    all_user_channels = client.list_channels_for_user_request(team_details.get('id', ''), user_id)
+
+    params: dict[Any, Any] = {}
+    list_private_channels = client.list_channel_request(team_details.get('id', ''), params, get_private=True)
```

Comment:
I want to present to you an issue that due to the fact that the endpoint of private channels supports receiving arguments of page and page size and the default value is passed, we might miss results and bring partial results of private channels because when intersecting between the user's channels list and the private channels list, there may be private channels that did not come from the endpoint. What do you think about it? Do you think there is a way to overcome this? Is this a plausible scenario?